### ISSUE TITLE
Silithus

### ISSUE THREAD

**Issue by thenumberpurplee** [SunwellWoW/Sunwell-TBC-Bugtracker#1939]
Decription: 
Silithus World buff is not working.

How it works: 
You click on a geyser in Silithus it does not  give you a buff called Silithyst you then run back to camp and turn it in and it doesnt give you the buff called Traces of Silithyst. 

How it should work: 
when you click the geyser it gives you the buff you then turn it in for the world buff

Source (you should point out proofs of your report, please give us some source):

**Comment by Jseay1313** [comment]
Just gonna put it out there that this is a bug and you are 100% correct, but the actual Geyser and quest for honor have maybe worked on 2 private servers ever.

**Comment by Wojtyas** [comment]
Indeed, buff (29534) should be applied with quest completions: 9422, 9419
